File: Patch_wang/Native/native_iOSPatchBackend/PatchDemo/ViewController.m
//
//  ViewController.m
//  PatchDemo
//
//  Created by mino on 2018/3/10.
//  Copyright © 2018年 mino. All rights reserved.
//

#import "ViewController.h"
#import <ReactiveCocoa/ReactiveCocoa.h>
@interface ViewController ()
@property (weak, nonatomic) IBOutlet UIButton *click;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [self clickIt];
    // Do any additional setup after loading the view, typically from a nib.
    
    self.name = @"Hehe";
    
    
}


-(void)clickIt{
    @weakify(self);
    [[self.click rac_signalForControlEvents:UIControlEventTouchUpInside]subscribeNext:^(id x) {
        @strongify(self);
        self.view.backgroundColor = [UIColor grayColor];
    }];
    
}





- (IBAction)backTo:(UIButton *)sender {
    
    [self dismissViewControllerAnimated: YES completion: nil];
    
     NSLog(@"name is %@", self.name);
}







- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: DEng/zb_best/zb_best/ViewController.swift
//
//  ViewController.swift
//  zb_best
//
//  Created by Jz D on 2018/6/7.
//  Copyright © 2018年 Jz D. All rights reserved.
//

import UIKit

import WebKit

class ViewController: UIViewController {

    
    var webView: WKWebView!
    
    
    override func loadView() {
        let webConfiguration = WKWebViewConfiguration()
        webView = WKWebView(frame: .zero, configuration: webConfiguration)
        webView.uiDelegate = self
        view = webView
    }
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        let myURL = URL(string: "http://m.ztbest.com")
        let myRequest = URLRequest(url: myURL!)
        webView.load(myRequest)
        
        
    }

    
    
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}


extension ViewController: WKUIDelegate{
    
    
}













